### DISCUSSION TITLE
How should we be submitting homeworks?

### DISCUSSION THREAD

**Discussion by RebeccaVillanueva** [cooperrc/advanced-dynamics#16] [General]
Namely, should we be submitting ipynb files or pdfs?

**Comment by SentientBiped** [comment]
Hi, this question was already answered in the Homework-00 Q&A forum, but just to save you some time, download it as HTML and then print that file to PDF, then submit that PDF to the huskyCT page.

**Comment by cooperrc** [comment]
pdf, thanks!
